lib/ProcessingManager/ProcessingManager.android.js

```diff
@@ -28,6 +30,20 @@ export class ProcessingManager {
       .then((res) => res.source);
   }
 
+  static pad(source: sourceType, options: padOptions): Promise<string> {
```

Comment:
How did you feel after writing non-type-safe code 😂 ?

Reply:
😕